app/boards/shields/splitkb_aurora_sofle/splitkb_aurora_sofle.keymap

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/ext_power.h>
+
+/ {
```

Comment:
This PCB could also use a chain-length guide like https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/kyria/kyria_rev3.keymap#L10 I assume

Reply:
Indeed. Added.